src/lsp/cobol_cfg/cfg_jumps.ml

```diff
@@ -0,0 +1,161 @@
+(******************************************************************************)
+(*                                                                            *)
+(*     Copyright (c) 2021-2024 OCamlPro SAS                                   *)
+(*                                                                            *)
+(*     All rights reserved.                                                   *)
+(*     This file is distributed under the terms of the                        *)
+(*     OCAMLPRO-NON-COMMERCIAL license.                                       *)
+(*                                                                            *)
+(******************************************************************************)
+
+open Cobol_unit
+open Cobol_unit.Types
+open Cobol_common.Visitor
+open Cobol_common.Srcloc.INFIX
+
+type qualname = Cobol_ptree.qualname
+
+type jumps =
+  | Go of qualname
+  | GoDepending of qualname
+  | Perform of qualname
+  | Call of Cobol_ptree.call_prefix
+  | Entry of Cobol_ptree.entry_stmt
+
+module Jumps = Set.Make(struct
+    type t = jumps
+    let compare j1 j2 =
+      let to_int = function
+        | Go _ -> 0
+        | GoDepending _ -> 1
+        | Perform _ -> 2
+        | Call _ -> 3
+        | Entry _ -> 4
+      in match j1, j2 with
+      | Go qn1, Go qn2
+      | GoDepending qn1, GoDepending qn2
+      | Perform qn1, Perform qn2 -> Cobol_ptree.compare_qualname qn1 qn2
+      | Call cp1, Call cp2 -> Cobol_ptree.compare_call_prefix cp1 cp2
+      | Entry en1 , Entry en2 -> Cobol_ptree.compare_entry_stmt en1 en2
+      | _ -> to_int j2 - to_int j1
```

Comment:
Why not use ppx-derivers for those? (as we used in other parts of the code)?

Reply:
Because I don't know how to use them :upside_down_face: